fairlearn/reductions/grid_search/grid_search.py

```diff
@@ -74,6 +77,10 @@ def accumulate_integer_grid(self, index, max_val):
                 self.accumulate_integer_grid(index+1, max_val-abs(current_value))
 
 
+_constraint_strings = {flc.demographic_parity: DemographicParity(),
+                       flc.bounded_group_loss: GroupLossMoment(ZeroOneLoss())}
+
```

Comment:
I don't think that ZeroOneLoss is a good default--and I'm not sure there's a good default here. I'd prefer to require an additional keyword argument "loss" defaulted to None, and would throw an error when constraints="bounded_group_loss", but loss is not set.

Reply:
I don't think that adding another keyword argument to the constructor to allow one of the other arguments to possibly be a string is a good pattern.